src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -0,0 +1,284 @@
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { copyArrayOfObjects, findComponent, uuid4 } from '@/utils/utils';
+import { DataTableFilter, DataTableFormElementFilters } from '@/constants/types';
+import {
+  ACCORDION_CLASSES,
+  BUTTON_CLASSES,
+  CHECKBOX_CLASSES,
+  FORM_CLASSES,
+  ICON_CLASS,
+  INPUT_CLASSES,
+  PORTAL_CLASS,
+  SELECT_CLASSES,
+  UTILITY_CLASSES,
+} from '@/constants/classes';
+import { compareFilters, getElementFilterData, updateFilterData } from '@/components/data-table-filters/FilterUtils';
+import { DATA_TABLE_EVENTS } from '@/constants/events';
+import DataTableFilters from '@/components/data-table-filters/DataTableFilters';
+
+Vue.config.ignoredElements = ['chi-popover'];
+// This element ignore is necessary as currently chi-popover is not a
+// Chi Vue component but a web component loaded from an external source
+
+// eslint-disable-next-line
+declare const chi: any;
+@Component
+export default class AdvancedFilters extends Vue {
+  @Prop() filtersData?: DataTableFilter[];
+
+  advancedFiltersData?: DataTableFilter[] = [];
+  _advancedFilterUuid?: string;
+  _advancedFilterButtonId?: string;
+  _advancedFilterPopoverId?: string;
+  _advancedFilterAccordionId?: string;
+  // eslint-disable-next-line
+  _advancedFiltersAccordion?: any;
+
+  created() {
+    this.advancedFiltersData = copyArrayOfObjects(this.$props.filtersData);
```

Comment:
I need to copy `advancedFiltersData` so that I can update it when advanced filters change and make the advancedFilters component rerender reactive to the change. Apply button `disabled` attribute depends on it